server/statistics/hot_peer_cache_test.go

```diff
@@ -67,19 +67,20 @@ const (
 )
 
 type testCacheCase struct {
-	kind     FlowKind
-	operator operator
-	expect   int
+	kind       FlowKind
+	operator   operator
+	expect     int
+	needDelete bool
 }
 
 func (t *testHotPeerCache) TestCache(c *C) {
 	tests := []*testCacheCase{
-		{ReadFlow, transferLeader, 2},
-		{ReadFlow, movePeer, 1},
-		{ReadFlow, addReplica, 1},
-		{WriteFlow, transferLeader, 3},
-		{WriteFlow, movePeer, 4},
-		{WriteFlow, addReplica, 4},
+		{ReadFlow, transferLeader, 3, false},
+		{ReadFlow, movePeer, 4, true},
+		{ReadFlow, addReplica, 4, false},
```

Comment:
why change them?

Reply:
The `CheckRegionFlow` used in test will create the hotstat for all the peers for both read and write now, thus the expectedItem count changed.